homeworks/nikita.karchinskiy_Nikkarchinskiy/5-rxjs/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Document</title>
+    </head>
+    <body>
+        <header class="header" data-header>
+            <div class="header-wrapper-l">
+                <a class="header-logo" href="#">Logo</a>
+                <nav class="header-nav">
+                    <menu class="menu-nav">
```

Comment:
You don't need the `menu` tag here. It's better to remove it:
w3schools.com/tags/tag_nav.asp